### ISSUE TITLE
Matter physics adding sprite as gameObject

### ISSUE THREAD

**Issue by RollinSafary** [photonstorm/phaser#4112]
## Version

* Phaser Version:Phaser 3.14

* Operating system:MacOS X 10.13.6

* Browser:Google Chrome Version 69.0.3497.100 (Official Build) (64-bit)

## Description

I've a sprite with following properties:

1. angle: 45 (any except default 0)
2. origin: {x: 0, y: 0} (any except default 0.5)
3. scale: {x: 10, y: 1} (any except default {x: 1, y: 1}

 After calling
```this.matter.add.gameObject(mySprite)```

My sprite will have

1. angle: 0 (default)
2. origin: {x: 0.5, y: 0.5} (default)
3. scale properties of body: {x: 1, y: 1}

## Example Test Code

```
var config = {
    type: Phaser.AUTO,
    width: 800,
    height: 600,
    backgroundColor: '#1b1464',
    parent: 'phaser-example',
    physics: {
        default: 'matter',
        matter: {
            debug: true,
            gravity: {
                x: 0,
                y: 0,
            }
        }
    },
    scene: {
        preload: preload,
        create: create
    }
};

var game = new Phaser.Game(config);

function preload ()
{
    this.load.image('blue', 'assets/sprites/columns-blue.png');
}

function create ()
{
    this.matter.world.setBounds().disableGravity();
    var rect = this.add.image(200, 50, 'blue');
    rect.setOrigin(0)
    rect.setAngle(45)
    rect.setScale(2)
    // rect = this.matter.add.gameObject(rect) //uncomment this line to see effect
}

```

## Additional Information

I know, that body is being created with size of texture, but I think, that body must get properties from gameObject for which it was created. Angle, scale, origin, e.t.c. Otherwise we will have a visible gameObject in one place and will expect some reaction, but it's body will be in another way.

**Comment by iamchristopher** [comment]
I ran into this as well. Refer to discussions on #4067 (specifically [this comment](https://github.com/photonstorm/phaser/issues/4067#issuecomment-424707345)).

TL;DR: The `x`, `y`, `rotation`, and `scale` properties of a Game Object are reset when you apply a Matter physics body and instead take on the properties of the physics body itself. This gives the net effect of "resetting" those properties.

**Comment by photonstorm** [comment]
It's true that the moment you create a Matter Game Object that the Game Object in question will inherit the properties of the Body and not the other way around. From this point on, the Game Object only reflects the Body values (Matter bodies do not use GO properties), so it made sense to have it this way around. If you want to provide default angle, position, etc then you can pass it in the Body Config when creating the Matter Game Object and the sprite will pick those up.
